angular-workspace/projects/ni/nimble-angular/src/directives/icon-base/nimble-icon-base.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+import type { Icon } from '@ni/nimble-components/dist/esm/icon-base';
+import type { IconSeverity } from '@ni/nimble-components/dist/esm/icon-base/types';
```

Comment:
Yep from public api since it should technically be exported from each icon but because we don't have entrypoints per component yet we can't re-export it multiple times like that:

https://github.com/ni/nimble/blob/5c613b867b6b034ea4ae96e562be5070be38b353/angular-workspace/projects/ni/nimble-angular/src/public-api.ts#L73-L77